README.tr.md

```diff
@@ -0,0 +1,85 @@
+# sway
+
+[English][en] - [日本語][ja] - [Français][fr] - [Українська][uk] - [Español][es] - [Polski][pl] - [中文-简体][zh-CN] - [Deutsch][de] - [Nederlands][nl] - [Русский][ru] - [中文-繁體][zh-TW] - [Português][pt] - [Danish][dk] - [한국어][ko] - [Română][ro] - **[Türkçe][tr]**
+
+Sway, [i3]-uyumlu bir [Wayland] dizgicisidir. [SSS][FAQ]'yi okuyun. 
+[IRC kanalı][IRC channel]na katılın \(irc.freenode.net'te #sway (İngilizce)).
```

Comment:
I don't know Turkish, but is there a missing space between `]` and `na`?

Reply:
-n is thematic vowel and -a is case suffix
So, the are compound to word but the the translation of [IRC channel] is [IRC kanalı]
So it looks like this
![resim](https://user-images.githubusercontent.com/60751519/116854711-ccbf5200-ac00-11eb-9edc-42659e568b5c.png)

Reply:
OK, thanks for the explanation!